querqy-core/src/main/java/querqy/model/builder/converter/MapConverter.java

```diff
@@ -13,17 +13,17 @@
 
 public class MapConverter {
 
-    private boolean parseBooleanToString = false;
-    private boolean includeNullValues = false;
+    public static final MapConverter DEFAULT_MAP_CONVERTER = new MapConverter(false, false);
+    public static final MapConverter MAP_CONVERTER_NULL_VALUES = new MapConverter(false, true);
+    public static final MapConverter MAP_CONVERTER_BOOL_STRING = new MapConverter(true, false);
+    public static final MapConverter MAP_CONVERTER_BOOL_STRING_NULL_VALUES = new MapConverter(true, true);
 
```

Comment:
Hmm, I am not really convinced of this. I think this is rather confusing for a user, as the user will not understand what "BOOL_STRING" means. I strongly favor the prior solution.